package_schemas/inputs_unaids_estimates/1_inputs_unaids_estimates.json

```diff
@@ -302,6 +302,26 @@
                 "preset": "hidden_value",
                 "field_value": "inputs-unaids-naomi-output-zip"
             }]
+        },
+        {
+            "name": "Naomi Results and Summary Report",
+            "about": "The Naomi HTML report file",
+            "resource_type": "inputs-unaids-naomi-report",
+            "resource_fields": [{
+                "field_name": "name",
+                "preset": "hidden_value",
+                "field_value": "Naomi Results and Summary Report"
+            }, {
+                "field_name": "format",
+                "label": "Format",
+                "preset": "resource_format_autocomplete",
+                "required": true,
+                "default": "URL"
```

Comment:
This could be right, though I'm just querying it - surprised that the format shouldn't be "HTML"

Reply:
Yes, I thought that it might work better if we set it to URL so that we can configure the link preview instead of downloading HTML file.

Reply:
Sure, then it sounds like you have reasoned through the idea.